src/calculateRentalCost.js

```diff
@@ -4,7 +4,26 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rentDay = 40;
+  let fullAmount = 0;
+
+  for (let i = 1; i <= days; i++) {
+    fullAmount += rentDay;
+  }
+
+  if (days >= 7) {
+    fullAmount = fullAmount - 50;
+
+    return fullAmount;
```

Comment:
Following the checklist item 4 and 5, you can simplify your if conditions by removing the 'else' and just returning the 'fullAmount' after the conditional blocks.